### PR TITLE
Add security section

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description  | Add security section. See below
| Fixed ticket | 

## Description

I add a Security section that
- replaces https://devdocs.prestashop-project.org/8/project/security-policy/ (I will create a redirect PR after this PR is merged)
- provides a page for the bug bounty
- announces the bug bounty is currently on pause

After this PR is merged I will update all places where we refer to the bug bounty to redirect them to this page.

### CODE REVIEW

File: content/security/security-policy.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: Security policy
+---
+
+# Security Policy
+
+Obviously, PrestaShop project security is a critical matter. PrestaShop teams are dedicated to keep a high level of security in every aspects of the software.
```
Comment (kpodemski) [inline content/security/security-policy.md:7]:
```suggestion
Obviously, PrestaShop project security is a critical matter. Project members are dedicated to keeping a high level of security in every aspect of the software.
```

File: content/security/security-release-process.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+title: Security release process
+---
+
+# Security release process
+
+Here is a short summary of the steps followed by the security team:
+
+1) Security issues is reported to security-core@prestashop.com or through the [Bug Bounty Program][bug-bounty].
+
+2) Security issues is assessed to identify their criticality level.
```
Comment (kpodemski) [inline content/security/security-release-process.md:11]:
```suggestion
2) Security issues are assessed to identify their criticality level.
```

File: content/security/security-release-process.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+title: Security release process
+---
+
+# Security release process
+
+Here is a short summary of the steps followed by the security team:
+
+1) Security issues is reported to security-core@prestashop.com or through the [Bug Bounty Program][bug-bounty].
```
Comment (kpodemski) [inline content/security/security-release-process.md:9]:
```suggestion
1) Security issues are reported to security-core@prestashop.com or through the [Bug Bounty Program][bug-bounty].
```

File: content/security/security-release-process.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+2) Security issues is assessed to identify their criticality level.
+
+- Minor issues are scoped to be fixed in the next scheduled minor release
+- Critical issues are scoped to be fixed as soon as possible
+
+3) For both minor and critical issues, a [GitHub Security Advisory](https://help.github.com/en/github/managing-security-vulnerabilities/creating-a-security-advisory
+) will be created to register the issue in GitHub CVE database.
+
+4) A [Private Security Fork](https://docs.github.com/en/github/managing-security-vulnerabilities/collaborating-in-a-temporary-private-fork-to-resolve-a-security-vulnerability) is used to prepare a patch Pull Request for the advisory. The Pull Request then reviewed and tested by QA.
```
Comment (kpodemski) [inline content/security/security-release-process.md:19]:
```suggestion
4) A [Private Security Fork](https://docs.github.com/en/github/managing-security-vulnerabilities/collaborating-in-a-temporary-private-fork-to-resolve-a-security-vulnerability) is used to prepare a patch Pull Request for the advisory. The Pull Request is then reviewed and tested by QA.
```

File: content/security/security-release-process.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+5) When all patch Pull Requests are ready (in the event that multiple issues are reported), they are all merged and a new patch release is built and delivered. Security Advisories are published and the vulnerabilities are disclosed in a Release Note, urging all PrestaShop users to upgrade in order to protect their shops.
```
Comment (kpodemski) [inline content/security/security-release-process.md:21]:
```suggestion
5) When all patch Pull Requests are ready (in the event that multiple issues are reported), they are all merged, and a new patch release is built and delivered. Security Advisories are published, and the vulnerabilities are disclosed in a Release Note, urging all PrestaShop users to upgrade in order to protect their shops.
```

Comment (kpodemski) [reply content/security/security-release-process.md:21]: and I think you forgot about the handy illustration we have in the devdocs?

**Review by kpodemski — COMMENTED** [review]
a few comments :)

File: content/security/security-release-process.md (hunk 5 above)
Comment (matks) [reply content/security/security-release-process.md:21]: Then I need to update it, it is outdated

**Review by matthieu-rolland — APPROVED** [review]

File: content/security/security-release-process.md (hunk 5 above)
Comment (matks) [reply content/security/security-release-process.md]: @kpodemski I merge this one to move forward but I do not forget about the image I will do it in a later PR

File: content/security/security-policy.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+## Reporting a Vulnerability
+
+Security issues can be reported by sending an email to security-core@prestashop.com or through our [Bug Bounty Program][bug-bounty], which will go to security team members.
+
+When the security team receives a security bug report, the report will be assigned to a primary handler. 
+This person will coordinate the fix and release process, involving the following steps:
+
+ - Confirm the problem and determine the affected versions.
+ - Audit code to find any potential similar problems.
```
Comment (tleon) [inline content/security/security-policy.md:19]:
```suggestion
 - Audit the code to find any potential similar problems.
```

File: content/security/security-policy.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+ - Prepare fixes for all releases still under maintenance. These fixes will be released as fast as possible.
+
+The security team will follow up with a response indicating the next steps in handling the report.
+
+If the issue is confirmed, the security team will keep you informed of the progress towards a fix and full announcement, and may ask for additional information or guidance.
```
Comment (tleon) [inline content/security/security-policy.md:24]:
```suggestion
If the issue is confirmed, the security team will keep you informed of the progress towards a fix, a full announcement, and may ask for additional information or guidance.
```

File: content/security/security-release-process.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+1) Security issues are reported to security-core@prestashop.com or through the [Bug Bounty Program][bug-bounty].
+
+2) Security issues are assessed to identify their criticality level.
+
+- Minor issues are scoped to be fixed in the next scheduled minor release
+- Critical issues are scoped to be fixed as soon as possible
+
+3) For both minor and critical issues, a [GitHub Security Advisory](https://help.github.com/en/github/managing-security-vulnerabilities/creating-a-security-advisory
+) will be created to register the issue in GitHub CVE database.
```
Comment (tleon) [inline content/security/security-release-process.md:17]:
```suggestion
) will be created to register the issue in GitHub's CVE database.
```

**Review by tleon — COMMENTED** [review]
just some suggestions ;)

**Comment by matks** [comment]
@tleon Done in https://github.com/PrestaShop/open-source/pull/192/commits/e9ebf3e583ce57c08c5b07f7e1e59f4a4d5bba56 😉

**Pull request merged**